generator/AWSPSGeneratorLib/HelpMaterials/Examples/ELB2/New-ELB2LoadBalancer.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$ipAddressType = New-Object  Amazon.ElasticLoadBalancingV2.IpAddressType('ipv4')
+$albScheme = New-Object Amazon.ElasticLoadBalancingV2.LoadBalancerSchemeEnum('internet-facing')
+$elbType = New-Object Amazon.ElasticLoadBalancingV2.LoadBalancerTypeEnum('Application')
+
+New-ELB2LoadBalancer -IpAddressType $ipAddressType -Name 'New-Test-ALB' -Scheme $albScheme -SecurityGroup 'sg-07c3414abb8811cbd' -subnet 'subnet-c37a67a6','subnet-fc02eea0' -Type $elbType -Region us-east-1
```

Comment:
Can be simplified to `-Type application -Scheme internet-facing -IpAddressType ipv4`